app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<p><%= @recipe.name %></p>
+<p><%= @recipe.complexity %></p>
+<p><%= @recipe.genre %></p>
+<div class="recipe_ingredients">
+  <% @recipe.ingredients.each do |ingredient| %>
+    <p><%= ingredient.name %></p>
+  <% end %>
+  Total Cost: <%= @recipe.total_ingredient_cost %>
+</div>
```

Comment:
Great job abstracting the `total_ingredient_cost` method to the Model over doing the AR in the view which would be an MVC infraction.